code/datums/mutantraces.dm

```diff
@@ -2019,6 +2079,25 @@
 			if ("milk")
 				if (mob.emote_check(voluntary))
 					.= release_milk()
+			if ("moo_p")
+				if (mob.emote_check(voluntary, 50))
+					. = list("<B>[mob]</B> moos proudly!", "<i>moos proudly</i>")
+			if ("moo_s")
+				if (mob.emote_check(voluntary, 50))
+					. = list("<B>[mob]</B> moos solemnly", "<i>moos solemnly</i>")
+			if ("moo_h")
+				if (mob.emote_check(voluntary, 50))
+					. = list("<B>[mob]</B> moos heartily", "<i>moos heartily</i>")
+			if ("stamp")
+				if (mob.emote_check(voluntary, 50))
+					. = list("<B>[mob]</B> stamps their hoove!", "<i>stamps their hoove!</i>")
```

Comment:
Hoof, or hooves if the cow's stamping both legs. Also consider adding a check to see if they have enough legs to stamp, and maybe also another check to see if they're somehow wearing shoes, so it could show up as something like "Human Person stamps their cowboy boot(s)!"